edx_proctoring/api.py

```diff
@@ -1132,9 +1101,15 @@ def create_exam_attempt(
         time_remaining_seconds=time_remaining_seconds,
     )
 
-    # Only create a verified name after the attempt is created, as we need the attempt ID
-    if should_create_verified_name:
-        _create_verified_name(user_id, full_name, profile_name, attempt.id, name_affirmation_service)
+    if taking_as_proctored:
+        # Emit signal for attempt creation
+        attempt_status_signal.send(
+            sender='edx_proctoring.create_exam_attempt',
+            attempt_id=attempt.id,
+            user_id=user_id,
+            status=attempt.status,
+            full_name=full_name,
+        )
```

Comment:
we lost the concept of "should create verified name" - is that logic expected to be in the name affirmation service now?

Reply:
Yes, that was my intention - have both the translation of statuses and updating/creating a verified name be controlled by name affirmation